### PR TITLE
DFP API cleanup

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Mostly moving code around, but I also removed `firstAdDisplayed` and `firstAdRendered` which never seemed to make sense. All `addSlot` does is add a slot to the queue, so as long as everything is sealed in `googletag.cmd` calls, we're on the safe side.

**LOL** a history of coincidences: later in the day, @JamieKMorrison asked why there's been a drop in MPU impressions on mobile. My theory is that inline ads are inserted via `addSlot` and that it is done _before_ `firstAdDisplayed` is set to true (which happens in [`instantLoad`](https://github.com/guardian/frontend/blob/master/static/src/javascripts-legacy/projects/commercial/modules/dfp/display-lazy-ads.js#L21)). Getting rid of that flag should solve the problem.

### CODE REVIEW

**Comment by regiskuckaertz** [comment]
[Cleaner front-end DFP API](https://trello.com/c/RUmfMhhM/276-cleaner-front-end-dfp-api)

**Comment by regiskuckaertz** [comment]
Hopefully the unspeakable thing I did to fix tests will be removed that the great work @katebee 's been doing in #16218

File: static/src/javascripts-legacy/projects/commercial/modules/dfp/dfp-env.js [hunk 1]
```diff
@@ -6,12 +6,6 @@ define([
         /* renderStartTime: integer. Point in time when DFP kicks in */
         renderStartTime: -1,
 
-        /* firstAdDisplayed: boolean. Set to true when the first advert is getting displayed (via googletag.display) */
-        firstAdDisplayed: false,
-
-        /* firstAdRendered: boolean. Set to true when the first advert has come back from DFP */
-        firstAdRendered: false,
```
Comment (rich-nguyen) [inline static/src/javascripts-legacy/projects/commercial/modules/dfp/dfp-env.js]: 👋

**Review by rich-nguyen — APPROVED** [review]

**Comment by rich-nguyen** [comment]
[Ad Ratio fell around 6 March (MPUs and mobile look most likely culprits)](https://trello.com/c/GCuwtwnQ/279-ad-ratio-fell-around-6-march-mpus-and-mobile-look-most-likely-culprits)

**Pull request merged**